### ISSUE TITLE
Free camera controls are not explained and don't seem to allow full freedom on tracks

### ISSUE THREAD

**Issue by wp9015362** [phoboslab/wipeout#7]
Hello,

the free camera controls in the viewer are not explained.

Any chance you could add a control scheme for the free camera to the viewer?

And the free camera also apparently does not seem to allow full freedom on tracks.

It looks like the mouse scroll wheel is assigned to zooming in and out.

However, when viewing a track, it is not possible to fully zoom in. At some point the zoom stops, even though it's still far away from the track.

Any chance you could fix that?

Regards

**Comment by Leonard2** [comment]
I agree the camera is pretty annoying. It should be a first-person point of view instead of rotating around an imaginary point.

**Comment by phoboslab** [comment]
Well, the Orbit Controls work nicely for the ship models etc, but I agree that navigating around the tracks is pretty tedious with this. A "first person" view would be less than ideal for the ship models. So maybe we need both.

For what it's worth, Orbit Controls are: Mouse-Left: Rotate, Mouse-Right: Pan, Scroll Wheel: Zoom
